public/static/covizu/_custom.js

```diff
@@ -0,0 +1,47 @@
+// customizations for virus-seq implementation
+
+// Increment the version here & in API only when there are breaking changes
+// to the data shape. This version number determines which data is loaded.
+const COVIZU_VERSION = '1.0.0';
+console.info('VirusSeq Covizu version', COVIZU_VERSION);
+
+// get API URL from URL query
+var urlParams = new URLSearchParams(window.location.search);
+var apiUrl = urlParams.get('apiUrl') || '';
```

Comment:
this seems to me like it could be something grabbed from global vars rather than the url (that way the app remains bookmarkable even if we change the API's url etc.)

Reply:
aahh! I see.

so, I take it you've checked if we injected the API's url into the app's index (using helmet or something like that), and then used `parent.whatever` within the iframe, it doesn't work? hmmm
I wonder whether there's something weird with that iframe. 🤷

Reply:
discussed on Slack - keeping it as is to avoid adding dependencies

Reply:
We didn't discuss avoiding dependencies... you said it didn't work, @samrichca.

Am I miss understanding something?

Reply:
check Slack :)

Reply:
Ok. Thanks for clarifying that for me!